src/Microsoft.VisualStudio.InteractiveWindow.Tests/InteractiveWindowTests.cs

```diff
@@ -44,11 +43,6 @@ void IDisposable.Dispose()
 
         private IInteractiveWindow Window => _testHost.Window;
 
-        private Task TaskRun(Action action)
```

Comment:
Why this change?

Reply:
A warning was reported for the name of `TaskRun`. Replacing with `Task.Run` seemed to be simpler than renaming once here only to remove it in the future.

Reply:
Removed this change and used a suppression instead.